src/Symfony/Component/Ldap/Tests/LdapClientTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Tests\Authentication\Provider;
+
+use Symfony\Component\Ldap\LdapClient;
+
+class LdapClientTest extends \PHPUnit_Framework_TestCase
+{
+
```

Comment:
Will be implemented, as we need to test the ldap_escape shiv.

At first, I actually wanted to implement this in a shim (in a php file), but I changed my mind and moved this in the LdapClient implementation.